uefi/firmwarefile.go

```diff
@@ -40,6 +40,27 @@ const (
 	fvFileTypeFFSMax   FVFileType = 0xFF
 )
 
+var supportedFiles map[FVFileType]bool
+
+func init() {
+	// These are the file types that we'll actually try to parse sections for.
```

Comment:
as discussed in chat, make this a statically defined map

Reply:
done